agent_code/auto_bomber/model.py

```diff
@@ -0,0 +1,49 @@
+import os
+import pickle
+import numpy as np
+import agent_code.auto_bomber.auto_bomber_config as config
+from agent_code.auto_bomber.transitions import Transitions
+
+
+class LinearAutoBomberModel:
+    def __init__(self, feature_extractor):
+        self.weights = None
+        self.feature_extractor = feature_extractor
+
+        if os.path.isfile(config.MODEL_PATH):
```

Comment:
Best practice in Python 3 is currently to use the object-oriented approach for handling paths, i.e. pathlib.Path.

See https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module for conversion table from os module to path lib module.

In our case I guess it is optional to migrate to the pathlib approach, just a suggestion from my side.

Reply:
Oh I see, this maybe even simplifies mocking, I will change this.
Seems that the template code where I copied the stuff from is also a bit dated